pyproject.toml

```diff
@@ -17,7 +17,6 @@ requires-python = ">=3.9"
 keywords = ["swxsoc", "nasa mission", "space weather"]
 classifiers = [
     "Development Status :: 3 - Alpha",
-    "License :: OSI Approved :: Apache 2.0",
```

Comment:
This was causing errors since this classifier is deprecated: https://packaging.python.org/en/latest/specifications/core-metadata/#classifier-multiple-use

We ran into this in the release build for `sammi`: https://github.com/swxsoc/sammi/actions/runs/11239835908/job/31247834769